src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: $calendar-max-width;
+
+  @each $name, $value in $days {
+    &--day_#{$name} > &__day:first-child {
```

```suggestion
    &--day_#{$name} :first-child {
```
it could be written easier